plugins/woocommerce-beta-tester/src/tools/data/resolvers.js

```diff
@@ -85,3 +86,17 @@ export function* getBlockTemplateLoggingThreshold() {
 		throw new Error( error );
 	}
 }
+
+export function* getComingSoonMode() {
+	const path = `${ API_NAMESPACE }/tools/get-force-coming-soon-mode/v1`;
+
+	try {
+		const response = yield apiFetch( {
+			path,
+			method: 'GET',
+		} );
+		yield setComingSoonMode( response );
```

Comment:
I suspect this might not set the correct value? Do we need to parse the response?

Reply:
We don't need to parse the response. The response should be a string. We can directly use it.

 I think I might build a wrong zip file. 😓  It's working fine locally. I'll test on JN and let you know.